frontends/precinct-scanner/src/screens/admin_screen.tsx

```diff
@@ -179,6 +210,34 @@ export function AdminScreen({
           </div>
         </Bar>
       </Absolute>
+      {isShowingToggleLiveModeWarningModal && (
+        <Modal
+          content={
+            <Prose>
+              <h1>Export Backup to switch to Test Mode</h1>
```

Comment:
I _think_ our headers, like our buttons, use title case, so "switch" should probably be capitalized